src/common/lib/DevTools.browser.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {createDevTools} from 'redux-devtools';
```

Comment:
have you tried: (Atom-Import-Sort)[https://atom.io/packages/atom-import-sort]

for example this can be result: 
```jsx
import './App.styl';

import Component from 'react-pure-render/component';
import React, {PropTypes} from 'react';
import {connect} from 'react-redux';

import mapDispatchToProps from '../../common/app/mapDispatchToProps';
import mapStateToProps from '../../common/app/mapStateToProps';
import DevTools from '../../common/lib/DevTools';
import Footer from './Footer.react';
import Header from './Header.react';
```